web/src/features/auth/slices/authenticationSlice.ts

```diff
@@ -86,10 +93,21 @@ export const decodeRoles = (token: string | undefined) => {
   }
 }
 
+export const fireStartsEnabled = (isAuthenticated: boolean, roles: string[]) => {
+  return isAuthenticated && roles.includes('hfi_set_fire_starts')
+}
+
+export const cypressAuthFinished = (): AppThunk => dispatch => {
```

Comment:
I'm assuming this is only ever called when running Cypress tests, in which case I'd argue that this should go somewhere other than this app-wide file, and instead into a Cypress-specific file

Reply:
It can't, it needs access to the authenticateFinished function and I don't want to expose that elsewhere

Reply:
Soooooo.... I think there are some things we can do here. I think it's more about what we're naming this. We're saying "using this hard coded token when we're not doing real authentication".  It's something that one could in theory use beyond just cypress. If I wanted to do local development, without keycloak (say keycloak is down, or something) - then toggling authentication is useful. Let's call it something else.

The other thing to consider is of the token should be in the code, or if we can read it from the environment. I think we could just make it an environment variable.

Reply:
I added `TEST_AUTH` in env (true, false or missing) for this. `AuthWrapper` checks for that or `window.Cypress` since env variables in Cypress are a pain.